src/elements/content-explorer/ContentExplorer.tsx

```diff
@@ -1614,23 +1614,19 @@ class ContentExplorer extends Component<ContentExplorerProps, State> {
 
     getMetadataViewProps = (): ContentExplorerProps['metadataViewProps'] => {
         const { metadataViewProps } = this.props;
-        const { tableProps } = metadataViewProps ?? {};
-        const { onSelectionChange } = tableProps ?? {};
+        const { onSelectionChange } = metadataViewProps ?? {};
         const { selectedItemIds } = this.state;
 
         return {
             ...metadataViewProps,
-            tableProps: {
-                ...tableProps,
-                selectedKeys: selectedItemIds,
-                onSelectionChange: (ids: Selection) => {
-                    onSelectionChange?.(ids);
-                    const isSelectionEmpty = ids !== 'all' && ids.size === 0;
-                    this.setState({
-                        selectedItemIds: ids,
-                        ...(isSelectionEmpty && { isMetadataSidePanelOpen: false }),
-                    });
-                },
+            selectedKeys: selectedItemIds,
```

Comment:
yea i think u might be duplicating efforts here? https://github.com/box/box-ui-elements/pull/4265/files#diff-a107a2b349a68fe6d161859e31869b4c18f7caa360a28e2d0a4c50d55fc012c1

maybe just rebase on top of his?